src/admin-views/troubleshooting/ea-status/server-connection.php

```diff
@@ -51,5 +51,5 @@
 		/>
 		<?php echo esc_html( $message ); ?>
 	</td>
-	<td><?php echo $notes;  // Escaping handled above. ?></td>
-</tr>
\ No newline at end of file
+	<td><?php echo wp_kses_post( $notes ); ?></td>
```

Comment:
We dont need to do this again. there was a comment about how the escaping is handled above. maybe add the ignore for PHPCS.